test/libmpv_test.c

```diff
@@ -271,9 +288,140 @@ static void test_options_and_properties(void)
         fail("Node: expected 1 but got %d'!\n", result_node.u.flag);
 }
 
+static void test_track_selection(char *directory)
```

Comment:
how about moving splitting this into a different file and test run?

Reply:
I considered that but this file has a lot of generic useful stuff in it for the test.

Reply:
I would move them to a shared file.